lib/fluent/plugin/out_mongo.rb

```diff
@@ -172,7 +172,6 @@ def client
     def collect_records(chunk)
       records = []
       chunk.msgpack_each {|time, record|
-        record[@time_key] = Time.at(time || record[@time_key]) if @include_time_key
```

Comment:
Hmm... can `inject_values_to_record` insert Time object to `time_key`?
The important point is `Time.at`.